site/docs/spark-configuration.md

```diff
@@ -94,6 +94,14 @@ Spark's built-in catalog supports existing v1 and v2 tables tracked in a Hive Me
 
 This configuration can use same Hive Metastore for both Iceberg and non-Iceberg tables.
 
+### Using catalog specific Hadoop configuration values
+
+Similar to configuring Hadoop properties by using `spark.hadoop.*`, it's possible to set per-catalog Hadoop configuration values when using Spark by adding the property for the catalog with the prefix `spark.sql.catalog.(catalog-name).hadoop.*`. These properties will take precedence over values configured globally using `spark.hadoop.*` and will only affect Iceberg tables.
+
+```plain
+spark.sql.catalog.hadoop_prod.hadoop.fs.s3a.endpoint = http://aws-local:9000
```

Comment:
Maybe add an example for `hadoop.hive.metastore.uris`, which is one of the most common use case here

Reply:
Sure. I will update to that instead.

Reply:
Sorry for late response, I thought I hit comment and I had not.

Wouldn't hive metastore uri's be set via the catalog's existing exposed `uri` parameter? E.g. `spark.sql.catalog.(catalog-name).uri`: https://github.com/apache/iceberg/blame/master/site/docs/spark-configuration.md#L60

Reply:
I'll put `hadoop.hive.metastore.kerberos.principal=hadoop/_HOST@REALM` possibly instead?

Reply:
Yes, I don't think that we want to point to the metastore URI because that's what our `uri` property overrides.